kernel/src/arch/x86/smp/smp.cpp

```diff
@@ -177,7 +177,171 @@ namespace smp
                     }
                 },
                 core_id);
-
+            proc::make_kthread_args(
+                +[](std::uint64_t arg) {
+                    klog::log("I got an argument and I don't know what it's for but here it is: %lu", arg);
+                    auto prev_msg = 0;
+                    while (true)
+                    {
+                        auto encoded_keypress = inb(0x60);
+                        if (encoded_keypress == prev_msg)
+                            continue;
+                        else
+                            prev_msg = encoded_keypress;
+                        // klog::log("Encoded Keypress: %02X, %02X", encoded_keypress, prev_msg);
+                        char us_qwerty_translation[] = {0,
```

Comment:
Turn the table into a parameter of the "read keystroke" function I described above. Also use `inline static constexpr` to declare constants including LUTs. Also make the array bounded so it's clear how big it should be.